doc/tech/glossary.md

```diff
@@ -20,15 +20,15 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 - **Network Function Virtualisation (NFV)**: The concept of separating network functions from the hardware they run on by using a virtual hardware abstraction layer.
 - **Network Function Virtualisation Infrastructure (NFVI)**: The totality of all hardware and software components used to build the environment in which a set of VAs are deployed (also referred to as cloud infrastructure).
- >_*Note:*_ The NFV-Infrastructure can span across many locations, e.g. places where data centres or edge nodes are operated. The network providing connectivity between these locations is regarded to be part of the cloud infrastructure. cloud infrastructure and VNF are the top-level conceptual entities in the scope of Network Function Virtualisation. All other components are sub-entities of these two main entities.
-
+ >_*Note:*_ The NFV-Infrastructure can span across many locations, e.g. places where data centres or edge nodes are operated. The network providing connectivity between these locations is regarded to be part of the cloud infrastructure. **Cloud Infrastructure** and **VNF** are the top-level conceptual entities in the scope of Network Function Virtualisation. All other components are sub-entities of these two main entities.
```

Comment:
* NFV-Infrastructure -> NFVI
* I think here we should keep NFVI instead of Cloud Infrastructure as we are referring to NFV.

Reply:
yes agree - have updated in https://github.com/cntt-n/CNTT/pull/1328/commits/00adbd31f04475399d99339a95c455aa1b700593